app/src/org/commcare/dalvik/application/CommCareApplication.java

```diff
@@ -783,10 +806,10 @@ public boolean matches(UserKeyRecord ukr) {
         //TODO: We can just delete the db entirely.
 
         Editor sharedPreferencesEditor = CommCareApplication._().getCurrentApp().getAppPreferences().edit();
+        sharedPreferencesEditor.putString(CommCarePreferences.LAST_LOGGED_IN_USER, null).commit();
 
-        sharedPreferencesEditor.putString(CommCarePreferences.LAST_LOGGED_IN_USER, null);
-
-        sharedPreferencesEditor.commit();
+        // manually clear file-backed fixture storage to ensure files are removed
+        CommCareApplication._().getFileBackedUserStorage("fixture", FormInstance.class).removeAll();

```

Comment:
If we wanted to do this in one swoop, instead of calling it on all tables that are file-backed, we could delete the `app/some_app_id/file_db/ukr_uuid_for_user` directory